contents/newsletter/how-to-get-job-startup.md

```diff
@@ -0,0 +1,111 @@
+---
+title: How to get a great job at a great startup
+date: 2024-07-18
+author:
+  - james-temperton
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/v1713521535/posthog.com/contents/blog/evolution-of-founders.jpg
+featuredImageType: full
+tags:
+  - People
+crosspost:
+  - Blog
+---
+
+We get a lot of job applications. In the last two months, we’ve had 900 people apply for engineering roles:
+
+* 86 got a first interview
+* 55 got a technical interview
+* 10 did a [SuperDay](https://posthog.com/handbook/people/hiring-process#4-posthog-superday) 
+* 4 were hired 
+
+![hiring engineers](https://res.cloudinary.com/dmukukwp6/image/upload/hiring_hogs_e172cf51da.png)
+
+That means we hired 0.4% of people who applied for engineering roles.
+
+So, how do you join the 0.4% of superstar engineers – not just at PostHog, but at any other fast-growing startup?
+
+## Think like a recruiter 
+
+First, some hard truths. There’s a huge gap between what job applicants and recruiters experience. With the exception of very early-stage companies, most startups will use an applicant tracking system like Ashby, Greenhouse, or Lever for hiring. These are designed for speed, not to make you look good. 
+
+You need to think about the information density of your application – less dense is good – and also avoid any formatting that the application system is likely to remove. Keep it simple.
+
+The same rules apply for your resume. Prioritize clarity over design. Black text, white background, no fancy graphics. And please, no profile pictures.
+
+![juraj application](https://res.cloudinary.com/dmukukwp6/image/upload/coverletters_9d4dbdb02e.png)
+
+### Brevity is your friend
+
+A startup recruiter will, if you’re lucky, spend more than a few seconds reading your application. Brevity and clarity are key to try and ensure your application isn’t quickly dismissed.
+
+We did the math and, on average, we spend less than one minute reviewing each application we receive. Automated or junk applications (hello, ChatGPT) will be discarded in less than five seconds. Truly great applications will be read for up to five minutes.
+
+### Spend time on a short cover letter
```

Comment:
I don't love this heading. It's kind of like "spend time doing thing". Could be more direct about what people should do here.